.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  push:
+    branches:
+    - main
+    - master
+    tags:
+    - 'v*'
+
+name: Release policy
+
+jobs:
+
+  test:
+    name: run tests and linters
+    uses: kubewarden/github-actions/.github/workflows/reusable-test-policy-rust.yml@v1
+
+  release:
+    needs: test
+    permissions:
+      # Required to push to GHCR
+      contents: write
```

Comment:
the current comment is somewhat incorrect, as `contents` is required for creating GH Releases. I would prefer something like this:
```
      # Required to create GH releases
      contents: write
      # Required to push to GHCR
      packages: write
      # Required by cosign keyless signing
      id-token: write
```

Reply:
fixed